pkgs/by-name/cr/crewai/package.nix

```
@@ -0,0 +1,10 @@
+{
+  python3Packages,
+  versionCheckHook,
+}:
+
+python3Packages.toPythonApplication python3Packages.crewai // {
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
+}
```

```suggestion
{ python3Packages }: with python3Packages; toPythonApplication crewai;
```
`versionCheckHook` already runs in the python derivation.